vault/config.go

```diff
@@ -38,6 +38,29 @@ func ConfigPath() (string, error) {
 	return file, nil
 }
 
+// CreateConfig will create the config directory and file if they do not exist
+func CreateConfig() (error) {
+	file, err := ConfigPath()
+	if err != nil {
+		return err
+	}
+	dir := filepath.Dir(file)
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		os.Mkdir(dir, 0755)
```

Comment:
0700 probably makes sense for this?

Reply:
@lox oops! Fixed in the latest push. Thanks!